<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="rootStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="PageMenu">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QListWidget" name="SettingsList">
         <item>
          <property name="text">
           <string>Provider</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::MediaPlaylistShuffle"/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Player</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::Phone"/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PagePanel">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QWidget" name="Header" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="BreadCrumb">
            <property name="text">
             <string>Bread Crumb</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="BackButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string> Back</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QStackedWidget" name="Pages">
         <widget class="QWidget" name="ProviderPage">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QWidget" name="widget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QToolButton" name="AddProviderButton">
                <property name="text">
                 <string>Add Provider</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="ProvidersList"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_5"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
